aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+You can add Swashbuckle with any of the following approaches:
+
+- From the Package Manager Console: 
+
+.. code-block:: bash
+
+    Install=Package Swashbuckle -Pre
+
+- Add Swashbuckle to project.json:
+
+.. code-block:: javascript
+
+    "Swashbuckle": "6.0.0-"
+
+- In Visual Studio:
+	- Right click your project in Solution Explorer > Manage NuGet Packages 
+	- Enter Swashbuckle in the search box
+	- Check "Include prerelease" 
+	- Set the Package source to nuget.org 
+	- Tap the Swashbuckle package and then tap Install 
+
+
+
+Add and configure Swagger to the middleware
+-------------------------------------------
+
+Add SwaggerGen to the services collection in the Configure method, and in the ConfigureServices method, enable the middleware for serving generated JSON document and the SwaggerUI. 
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs
+    :language: c#
+    :start-after: snippet_Configure
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 13,44,47
+
+
+In Visual Studio, press ^F5 to launch the app and navigate to ``http://localhost:<random_port>/swagger/v1/swagger.json`` to see the document generated that describes the endpoints. 
+
+.. note:: Microsoft Edge, Google Chrome and Firefox display JSON documents natively.  There are extensions for Chrome that will format the document for easier reading. *Example below reduced for brevity.*
+
+.. code-block:: javascript
+
+    {
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
+        "title": "API V1"
+    },
+    "basePath": "/",
+    "paths": {
+        "/api/Todo": {
+        "get": {
+            "tags": [
+            "Todo"
+            ],
+            "operationId": "ApiTodoGet",
+            "consumes": [],
+            "produces": [
+            "text/plain",
+            "application/json",
+            "text/json"
+            ],
+            "responses": {
+            "200": {
+                "description": "OK",
+                "schema": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/TodoItem"
+                }
+                }
+            }
+            },
+            "deprecated": false
+        },
+        "post": {
+            ...
+        }
+        },
+        "/api/Todo/{id}": {
+        "get": {
+            ...
+        },
+        "put": {
+            ...
+        },
+        "delete": {
+            ...
+    },
+    "definitions": {
+        "TodoItem": {
+        "type": "object",
+        "properties": {
+            "key": {
+            "type": "string"
+            },
+            "name": {
+            "type": "string"
+            },
+            "isComplete": {
+            "type": "boolean"
+            }
+        }
+        }
+    },
+    "securityDefinitions": {}
+    }
+
+This document is used to drive the Swagger UI which can be viewed by navigating to ``http://localhost:<random_port>/swagger/ui``
```

Comment:
@Rick-Anderson there is no "Show/Hide" functionality in a code snippet. I put in the `...` because you stated that this was too much. This is a generated file from Swagger, it is important to show this information as it is what drives the UI, it is also sort of a "WSDL". I added in the *Note* above this that the file was reduced for brevity.